optuna/dashboard.py

```diff
@@ -27,6 +27,20 @@
     import bokeh.themes
     import tornado.gen
 
+if _imports.is_successful():
```

Comment:
Just wondering but how come we're deferring the `bokeh` imports above but not the version check here?

Reply:
That was because `ImportError.name` was `None`, and the error message was not shown as expected. I input the `name` argument and move the version check into with-try_import clause in https://github.com/optuna/optuna/pull/1326/commits/1b7dcab2046e7f00dcf335840c7c9f00d23a3f9a.

```console
% optuna dashboard --study-name parallel --storage sqlite:///lgbtuner.db
[E 2020-06-08 15:55:34,522] Tried to import 'None' but failed. Please make sure that the package is installed correctly to use this feature. Actual error: Your version of bokeh is 2.0.2 . Please install bokeh version earlier than 2.0.0. Bokeh can be installed by executing `$ pip install 'bokeh<2.0.0'`. For further information, please refer to the installation guide of bokeh. .
```

Reply:
Ah, thanks for the explanation and your suggestion for fix. That totally slipped through my mind. Let me create a fix that for the similar `plotly` version check.